syntax = "proto3";

package demo;

service DemoService {
  rpc ProcessRequest (Request) returns (Response);
}

message Request {
  int32 jobId = 1;
  string query = 2;
}

message Response {
  int32 jobId = 1;
  string result = 2;
  bool success = 3;
}